src/nav/infrastructure.yml

```diff
@@ -176,6 +188,8 @@ pages:
             path: /docs/infrastructure/host-integrations/host-integrations-list/hashicorp-consul-monitoring-integration
           - title: IBM MQ monitoring integration
             path: /install/ibm-mq
+          - title: Monitor Jenkins with OpenTelemetry 
```

Comment:
Another one that seems oddly out of alphabetical

Reply:
@paperclypse I removed "Monitor."